authors/apps/articles/relations.py

```diff
@@ -0,0 +1,28 @@
+import re
+from rest_framework import serializers
+from authors.apps.articles.models import Tags
+from .messages import error_msgs
+from rest_framework.exceptions import ValidationError
+
+
+class TagsRelation(serializers.RelatedField):
+    """This class overwrites the serializer class for tags
+    to enable tags to be saved on a separate table when
+    creating an article
+    """
+
+    def get_queryset(self):
```

Comment:
for consistency include comments for this as well and the one below

Reply:
Okay thanks. Let me work on that

Reply:
I have worked on it. You can kindly review and resolve the conversation